cg/meta/deliver_ticket.py

```diff
@@ -91,6 +91,36 @@ def remove_files(self, reads: List[Path]) -> None:
             LOG.info("Removing file: %s", file)
             file.unlink()
 
+    def get_all_samples_from_ticket(self, ticket_id: int) -> list:
+        all_samples = []
+        cases: List[models.Family] = self.get_all_cases_from_ticket(ticket_id=ticket_id)
+        for case in cases:
+            for link_obj in case.links:
+                all_samples.append(link_obj.sample.name)
+        return all_samples
```

Comment:
similar functionality exists:
https://github.com/Clinical-Genomics/cg/blob/7b5366430ec63860dcbb1ed2df236af9707710f5/cg/store/api/findbusinessdata.py#L223

if its not good enough, you could create one in the store api